terraform/msk-plugin.tf

```diff
@@ -0,0 +1,7 @@
+module "msk-plugin" {
+  source      = "github.com/NetDocs-Apps/tf-msk-connector-mod-iac?v0.1//mod/msk-plugin"
+  tags        = var.tags
+  env         = var.env
+  file_name   = "kafka-connect-couchbase-4.2.4-SNAPSHOT.zip"
```

Comment:
If you pull latest from the `master` branch this will change to `"kafka-connect-couchbase-4.2.5-SNAPSHOT.zip"`.